modules/rhel-preparing-playbook-machine.adoc

```diff
@@ -56,7 +61,19 @@ If you use SSH key-based authentication, you must manage the key with an SSH age
 # subscription-manager attach --pool=<pool_id>
 ----
 
-. Enable the repositories required by {product-title} {product-version}:
+. Enable the repositories required by {product-title} {product-version}.
+.. On a {op-system-base} 8 system:
```

Comment:
See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output for this repo's guidance on formatting commands. Relevant:

> If a step in a procedure is to run a command, make sure that the step text includes an explicit instruction to "run" or "enter" the command.